Aplicacion/app/src/main/java/e/asus/whitedoc/ActividadProgramarCita.java

```diff
@@ -0,0 +1,102 @@
+package e.asus.whitedoc;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.content.pm.LabeledIntent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+import android.widget.Toast;
+
+import com.google.firebase.database.ChildEventListener;
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+
+import java.util.ArrayList;
+
+import model.User;
+
+public class ActividadProgramarCita extends AppCompatActivity {
+
+    ListView myListView;
+    ArrayList<String> myarrayList=new ArrayList<>();
+    //ArrayList<String> myEmailList=new ArrayList<>();
+    DatabaseReference mRef;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_actividad_programar_cita);
+
+        final ArrayAdapter<String> myArrayAdapter=new ArrayAdapter<String>(ActividadProgramarCita.this, android.R.layout.simple_list_item_1,myarrayList);
```

Comment:
Esto mal
```suggestion
    ListView myListView;
    ArrayList<String> myarrayList=new ArrayList<>();
    //ArrayList<String> myEmailList=new ArrayList<>();
    DatabaseReference mRef;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_actividad_programar_cita);

        final ArrayAdapter<String> myArrayAdapter=new ArrayAdapter<String>(ActividadProgramarCita.this, android.R.layout.simple_list_item_1,myarrayList);
```
Esto bien